source/_patterns/01-elements/buttons/button.scss

```diff
@@ -5,17 +5,18 @@
 .elm-button {
 	// Including a loading indicator
 	--loadingIndicator---color: currentColor;
+
+	$icon-size: var(--db-icon-size, 20);
 	align-items: center; // Centering the content vertically and horizontally
 
-	// Centering the content vertically and horizontally
-	align-items: center;
 	border: 1px solid transparent;
 	border-radius: 4px;
 	display: inline-flex;
 
 	font-weight: 700;
 	justify-content: center;
-	padding: to-rem($pxValue: 7) to-rem($pxValue: 16);
+	min-height: $db-sizing-md;
```

Comment:
@nmerget the medium sized button actually has a height of 2.5rem according to Zeplin.

Reply:
We are in a functional context: https://github.com/db-ui/core/blob/refactor-dbux-3-button-new/source/css/enterprise/db-ui-core.scss#L14

Therefore the height is 2 rem

Reply:
got it and aligned previously. Thanks a lot.